docs/madr/decisions/073-spiffe-compliance.md

```diff
@@ -0,0 +1,95 @@
+# SPIFFE Compliance
+
+* Status: accepted
+
+Technical Story: https://github.com/kumahq/kuma/issues/13659
+
+## Context and Problem Statement
+
+In today’s service mesh systems, secure communication between services is essential. To make that possible, each service needs a clear and trusted identity: so we know exactly who is talking to whom.
+
+One of the most trusted standards for defining these service identities is SPIFFE (Secure Production Identity Framework for Everyone). It gives us:
+
+* A standard way to name services: `spiffe://<trust-domain>/<path>`
+* A strict format for certificates, especially the Subject Alternative Name (SAN) - which must contain exactly one URI (the SPIFFE ID)
+* Tools and rules for issuing, rotating, and checking these identities
+* SPIFFE certificates identify workload, not an endpoint or API
+
```

Comment:
I don't fully understand this sentence here with the above context. Could you please explain some details?

Reply:
> SPIFFE certificates identify workload, not an endpoint or API

Each application might expose multiple endpoints. Currently, we create certificate for dataplane but we add identities of all endpoints to this certificate as SAN. That is not a good way, and certificate should just identify the workload and not each certificate for a separate API endpoint.